vars/coverityPost.groovy

```diff
@@ -0,0 +1,60 @@
+// vars/coverityPost.groovy
+
+  /**
+   * coverityPost step method
+   *
+   * @param config Map of parameters passed
+   *
+   * config['arch']                CPU architecture.  Default is blank.
+   *
+   * config['condition']           Name of the post step to run. 
+   *                               Rrequired to be one of
+   *                               'success', or 'unsuccessful'.
+   *
+   * config['success_script']      Script to run on successful build.
+   *                               Default 'ci/coverity_success.sh'.
+   *
+   * config['unsuccessful_script'] Script to run if build is not successful.
+   *                               Default 'ci/coverity_unsuccessful.sh'
+   *
+   * config['coverity_tarball']    Tarball to archive for coverity.
+   *                               default: 'coverity/*.tgz'
+   */
+
+def call(Map config = [:]) {
+
+  def arch = 'arch='
+  if (config['arch']) {
+    arch='arch=' + config['arch'] + ' '
+  }
+
+  if (config['condition'] == 'success') {
+
+    def success_script = config.get('success_script',
+                                    'ci/coverity_success.sh')
+    sh label: 'Coverity success',
+       script: arch + success_script
```

Comment:
You need a space here because if line 28 is not executed you end up with:
```
arch=ci/coverity_success.sh
```
```suggestion
       script: arch + ' ' + success_script      
```